packages/statemanager/test/stateManager.storage.spec.ts

```diff
@@ -20,133 +20,135 @@ describe('StateManager -> Storage', () => {
     { deactivate: true },
     { deactivate: false, size: 0 },
   ]) {
-    it(`should dump storage`, async () => {
-      const stateManager = new DefaultStateManager({ storageCacheOpts })
-      const address = new Address(hexToBytes('0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b'))
-      const account = createAccount()
-
-      await stateManager.putAccount(address, account)
-
-      const key = hexToBytes('0x1234567890123456789012345678901234567890123456789012345678901234')
-      const value = hexToBytes('0x0a') // We used this value as its RLP encoding is also 0a
-      await stateManager.putContractStorage(address, key, value)
-
-      const data = await stateManager.dumpStorage(address)
-      const expect = { [bytesToHex(keccak256(key))]: '0x0a' }
-      assert.deepEqual(data, expect, 'should dump storage value')
-    })
-
-    it("should validate the key's length when modifying a contract's storage", async () => {
-      const stateManager = new DefaultStateManager({ storageCacheOpts })
-      const address = new Address(hexToBytes('0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b'))
-      const account = createAccount()
-      await stateManager.putAccount(address, account)
-
-      try {
-        await stateManager.putContractStorage(address, new Uint8Array(12), hexToBytes('0x1231'))
-      } catch (e: any) {
-        assert.equal(e.message, 'Storage key must be 32 bytes long')
-        return
-      }
-
-      assert.fail('Should have failed')
-    })
-
-    it("should validate the key's length when reading a contract's storage", async () => {
-      const stateManager = new DefaultStateManager({ storageCacheOpts })
-      const address = new Address(hexToBytes('0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b'))
-      const account = createAccount()
-      await stateManager.putAccount(address, account)
-
-      try {
-        await stateManager.getContractStorage(address, new Uint8Array(12))
-      } catch (e: any) {
-        assert.equal(e.message, 'Storage key must be 32 bytes long')
-        return
-      }
-
-      assert.fail('Should have failed')
-    })
-
-    it(`should throw on storage values larger than 32 bytes`, async () => {
-      const stateManager = new DefaultStateManager({ storageCacheOpts })
-      const address = Address.zero()
-      const account = createAccount()
-      await stateManager.putAccount(address, account)
-
-      const key = zeros(32)
-      const value = hexToBytes('0x' + 'aa'.repeat(33))
-      try {
+    for (const prefixStorageTrieKeys of [false, true]) {
```

Comment:
This is the only changed line here (lol).

Diff is showing up as somewhat crazy. Don't get confused! 😆